src/WebJobs.Script/Rpc/LanguageWorkerChannelUtilities.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Converters;
+
+namespace Microsoft.Azure.WebJobs.Script.Rpc
+{
+    public static class LanguageWorkerChannelUtilities
+    {
+        private static int maxNumberOfErrorMessages = 3;
+
+        private static JsonSerializerSettings verboseSerializerSettings = new JsonSerializerSettings()
+        {
+            Formatting = Formatting.Indented,
+            NullValueHandling = NullValueHandling.Ignore,
+            Converters = new List<JsonConverter>()
+                    {
+                        new StringEnumConverter()
+                    }
+        };
+
+        public static JsonSerializerSettings VerboseSerializerSettings { get => verboseSerializerSettings; set => verboseSerializerSettings = value; }
+
+        public static bool IsLanguageWorkerConsoleLog(string msg)
```

Comment:
What is this for?

Reply:
this is used for filtering logs coming from stdout/stderr language worker console logs vs stdout/stderr logs coming from users code